hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -258,6 +259,14 @@
   public static final String SPECIAL_RECOVERED_EDITS_DIR =
     "hbase.hregion.special.recovered.edits.dir";
 
+  /**
+   * Whether to use {@link MetaCellComparator} even if we are not meta region. Used when creating
+   * master local region.
+   */
+  public static final String USE_META_CELL_COMPARATOR = "hbase.region.use.meta.cell.comparator";
```

Comment:
It's a little nit-pick, but I like my configurations to specify components via `.`-separator, and use `_` for component names. So this might be `hbase.master.region.use_meta_cell_comparator`.

Of course, this might be conflicting with other configuration names and uses. Would need to look things over to see what else is using the `"hbase.master"` namespace.